polaris/evaluate/_metric.py

```diff
@@ -30,6 +31,30 @@ def spearman(y_true: np.ndarray, y_pred: np.ndarray):
     return stats.spearmanr(y_true, y_pred).statistic
 
 
+def absolute_average_fold_error(y_true: np.ndarray, y_pred: np.ndarray):
```

Comment:
Let's add this to our API docs. You can add the following to `docs/api/evaluation.md`:

```

::: polaris.evaluate._metric.absolute_average_fold_error

---
```